autogen.sh

```diff
@@ -8,5 +8,5 @@ autoreconf -v -f --install || exit 1
 	--enable-maintainer-mode \
 	--disable-shared \
 	--without-libpam \
-	--with-selinux \
```

Comment:
You talk about systemd but you disable selinux here so I'm confused what the issue is.

Reply:
😂.

Reply:
selinux not needed.

Reply:
We're confused what you're actually trying to do.  Your commit message only and repeatedly mentioned systemd, not selinux, but you don't change anything to do with systemd.  Did a chunk of your patch get lost?